<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow{
background-color: qlineargradient(spread:pad, x1:0.107, y1:0.119455, x2:1, y2:1, stop:0 rgba(125, 93, 255, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="takePhoto_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  display: inline-block;
  border-radius: 4px;
  background-color:#ffffff;
  border: none;
  color: rgb(143, 143, 143);
  text-align: center;
  font-size: 28px;
  padding: 20px;
  width: 200px;
  transition: all 0.5s;
  cursor: pointer;
  margin: 5px;
}

QPushButton span {
  cursor: pointer;
  display: inline-block;
  position: relative;
  transition: 0.5s;
}

.button span:after {
  content: '\00bb';
  position: absolute;
  opacity: 0;
  top: 0;
  right: -20px;
  transition: 0.5s;
}

QPushButton:hover span {
  padding-right: 25px;
}

QPushButton:hover span:after {
  opacity: 1;
  right: 0;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>facial-recognition-scan.png</normaloff>facial-recognition-scan.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="trainface_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  display: inline-block;
  border-radius: 4px;
  background-color:#ffffff;
  border: none;
  color: rgb(143, 143, 143);
  text-align: center;
  font-size: 28px;
  padding: 20px;
  width: 200px;
  transition: all 0.5s;
  cursor: pointer;
  margin: 5px;
}

QPushButton span {
  cursor: pointer;
  display: inline-block;
  position: relative;
  transition: 0.5s;
}

.button span:after {
  content: '\00bb';
  position: absolute;
  opacity: 0;
  top: 0;
  right: -20px;
  transition: 0.5s;
}

QPushButton:hover span {
  padding-right: 25px;
}

QPushButton:hover span:after {
  opacity: 1;
  right: 0;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>591744.png</normaloff>591744.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>981</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget#tabWidget_2{
background-color: qlineargradient(spread:pad, x1:0.107, y1:0.119455, x2:1, y2:1, stop:0 rgba(125, 93, 255, 255), stop:1 rgba(255, 255, 255, 255))
}

QTabWidget::pane &gt; QWidget {
    background-color:  qlineargradient(spread:pad, x1:0.107, y1:0.119455, x2:1, y2:1, stop:0 rgba(125, 93, 255, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>View Attendance</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1011</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane &gt; QWidget {
    background-color: #262626;
}</string>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedKingdom"/>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <row>
       <property name="text">
        <string>Student</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Identification</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Full Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Course </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Gender</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Attendance</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Add/Search/Delete Student</string>
     </attribute>
     <widget class="QLineEdit" name="searchLineEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
font-size: 18px;
background:transparent;
border:none;
color:#717072

}</string>
      </property>
      <property name="text">
       <string>Search box</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idNumber">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>70</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
font-size: 18px;
background:transparent;
border:none;
color:#717072

}</string>
      </property>
      <property name="text">
       <string>Identification Input Box</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_gender">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>260</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
font-size: 18px;
background:transparent;
border:none;
color:#717072

}</string>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitButton">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>360</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  display: inline-block;
  border-radius: 4px;
  background-color:#ffffff;
  border: none;
  color: rgb(143, 143, 143);
  text-align: center;
  font-size: 28px;
  padding: 20px;
  width: 200px;
  transition: all 0.5s;
  cursor: pointer;
  margin: 5px;
}

QPushButton span {
  cursor: pointer;
  display: inline-block;
  position: relative;
  transition: 0.5s;
}

.button span:after {
  content: '\00bb';
  position: absolute;
  opacity: 0;
  top: 0;
  right: -20px;
  transition: 0.5s;
}

QPushButton:hover span {
  padding-right: 25px;
}

QPushButton:hover span:after {
  opacity: 1;
  right: 0;
}</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>130</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
font-size: 18px;
background:transparent;
border:none;
color:#717072

}</string>
      </property>
      <property name="text">
       <string>Full Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>360</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  display: inline-block;
  border-radius: 4px;
  background-color:#ffffff;
  border: none;
  color: rgb(143, 143, 143);
  text-align: center;
  font-size: 28px;
  padding: 20px;
  width: 200px;
  transition: all 0.5s;
  cursor: pointer;
  margin: 5px;
}

QPushButton span {
  cursor: pointer;
  display: inline-block;
  position: relative;
  transition: 0.5s;
}

.button span:after {
  content: '\00bb';
  position: absolute;
  opacity: 0;
  top: 0;
  right: -20px;
  transition: 0.5s;
}

QPushButton:hover span {
  padding-right: 25px;
}

QPushButton:hover span:after {
  opacity: 1;
  right: 0;
}</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_degree">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>200</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
font-size: 18px;
background:transparent;
border:none;
color:#717072

}</string>
      </property>
      <property name="text">
       <string>Students Course</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetTwo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>500</width>
        <height>400</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>250</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="gridStyle">
       <enum>Qt::DotLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Full Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Major</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Gender</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  display: inline-block;
  border-radius: 4px;
  background-color:#ffffff;
  border: none;
  color: rgb(143, 143, 143);
  text-align: center;
  font-size: 28px;
  padding: 20px;
  width: 200px;
  transition: all 0.5s;
  cursor: pointer;
  margin: 5px;
}

QPushButton span {
  cursor: pointer;
  display: inline-block;
  position: relative;
  transition: 0.5s;
}

.button span:after {
  content: '\00bb';
  position: absolute;
  opacity: 0;
  top: 0;
  right: -20px;
  transition: 0.5s;
}

QPushButton:hover span {
  padding-right: 25px;
}

QPushButton:hover span:after {
  opacity: 1;
  right: 0;
}</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255); font-size:80pt;
font: 8pt &quot;MV Boli&quot;;
font: 75 30pt &quot;Bahnschrift SemiCondensed&quot;;</string>
    </property>
    <property name="text">
     <string>Management System</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  display: inline-block;
  border-radius: 4px;
  background-color:#ffffff;
  border: none;
  color: rgb(143, 143, 143);
  text-align: center;
  font-size: 28px;
  padding: 20px;
  width: 200px;
  transition: all 0.5s;
  cursor: pointer;
  margin: 5px;
}

QPushButton span {
  cursor: pointer;
  display: inline-block;
  position: relative;
  transition: 0.5s;
}

.button span:after {
  content: '\00bb';
  position: absolute;
  opacity: 0;
  top: 0;
  right: -20px;
  transition: 0.5s;
}

QPushButton:hover span {
  padding-right: 25px;
}

QPushButton:hover span:after {
  opacity: 1;
  right: 0;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>90417.png</normaloff>90417.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Build-Library-Management-System-Python-PyQt5-master/icons.qrc"/>
 </resources>
 <connections/>
</ui>
